README.md

```diff
@@ -2,7 +2,21 @@
 
 A best-effort reimplementation of the Minecraft Forge API.
 
-## Contribution Notes
+## How do I use this?
 
-We use [MinecraftForge @ d28cd0352b6b0fe86e062f29e681c3b14572c6d5](https://github.com/MinecraftForge/MinecraftForge/tree/d28cd0352b6b0fe86e062f29e681c3b14572c6d5) as our reference commit. All features should be based on this version of Forge
-for consistency. This will be changed once we update to 1.15.
+Patchwork is not yet ready for general use. However, if you're ambitious and want to test, we provide prebuilt releases on [GitHub](https://github.com/PatchworkMC/patchwork-api/releases). Note that you will need to pass any Forge mods through [Patchwork Patcher](https://github.com/PatchworkMC/patchwork-patcher/releases) as well. While we do not expect most mods to work, bug reports would be appreciated.
+
```

Comment:
add a link to our discord here and note that there's more information about compatibility there